google-cloud-spanner/pom.xml

```diff
@@ -142,6 +142,16 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-protobuf</artifactId>
     </dependency>
+    <dependency>
```

Comment:
It seems that this change is now causing this problem further down the line with the `flatten-maven-plugin`: https://github.com/googleapis/java-spanner-jdbc/pull/293#issuecomment-742471649